tests/plotting/test_plotting.py

```diff
@@ -2468,6 +2468,20 @@ def test_add_text():
     plotter.show()
 
 
+@pytest.mark.skipif(
+    vtk.vtkMathTextFreeTypeTextRenderer().MathTextIsSupported(),
+    reason='Math text is not supported.',
+)
```

Comment:
This is needed for the conda job due to https://github.com/conda-forge/vtk-feedstock/issues/79